controllers/employeeDashboard.controller.js

```diff
@@ -0,0 +1,61 @@
+const User = require("./../modules/user.module");
+const {
+    setHomeAddress,
+    setVacationRequest,
+    getUserById,
+    getAllUsers,
+} = require("./../DAL/user.DAL");
+const bcrypt = require("bcrypt");
+const _ = require("lodash");
+const {
+    invalidId,
+    missing_homeAddress,
+} = require("./handleControllerErrors");
+
+
+findAllEmployees = (req, res) => {
+    const data = getAllUsers();
+    res.set('Content-Type', 'application/json');
+    res.writeHeader(200);
+    res.end(JSON.stringify(data));
```

Comment:
you don't need the content type here, we set it in the server.
and use a more common syntax:
res.status(200).json(JSON.stringify(data))